tasks/image-upscale/__init__.py

```
@@ -0,0 +1,71 @@
+        start_response = requests.post(start_url, json=start_payload,headers=headers)
+        start_response.raise_for_status()
+        request_id = start_response.json().get("request_id")
+        
+        if not request_id:
+            raise ValueError("No request_id received from start task")
+        
+        # Step 2: Check task status
+        status_url = f"https://console.oomol.com/api/tasks/fal/images/upscale/status/{request_id}"
+        
+        while True:
+            status_response = requests.get(status_url,headers=headers)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)